docs/version0.25.md

```diff
@@ -82,6 +83,11 @@ WARNING: All illegal access operations will be denied in a future release
 
 The `-verbose:module` option is now supported for Java 11 and later releases. This option writes information to `stderr` for each module that is loaded and unloaded.
 
+### Change the default `zlib` library on AIX&reg;
+
+OpenJ9 for AIX&reg; will use system `zlib` library by default instead of the bundled copy.
```

Comment:
"will use" -> "uses". Jere and below.
change "the" to "a". "the" implies there still is a bundled copy.

Reply:
Fixed in 337fa8b